src/strategies/winr-staking/index.ts

```diff
@@ -0,0 +1,92 @@
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+import { BigNumber } from '@ethersproject/bignumber';
+
+export const author = 'flushjb';
+export const version = '0.1.0';
+
+const abi = [
+  'function getActiveIndexes(address staker) external view returns (uint256[])',
+  'function vestingStakedAmount(address _account, uint256[] _indexes) external view returns (uint256 _totalStaked)',
+  'function getDividendStake(address _account, bool _isVested) external view returns (uint256 amount, uint256 profitDebt, uint256 weight, uint128 depositTime)'
+];
+
+// 'function getActiveIndexes(address staker) external view returns (uint256[])',
+
+type VotingPowers = Record<string, number>;
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses: string[],
+  options,
+  snapshot
+): Promise<VotingPowers> {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const multicaller = new Multicaller(network, provider, abi, { blockTag });
+
+  // Get each user's active indexes and dividend stakes
+  await Promise.all(
+    addresses.map(async (address) => {
+      await multicaller.call(
+        `activeIndexes.${address}`,
+        options.address,
+        'getActiveIndexes',
+        [address]
+      );
+      await multicaller.call(
+        `dividendStakes.${address}`,
+        options.address,
+        'getDividendStake',
+        [address, true]
+      );
+      await multicaller.call(
+        `dividendStakesVested.${address}`,
+        options.address,
+        'getDividendStake',
+        [address, false]
+      );
+    })
+  );
```

Comment:
Hi @flushjb we don't need await for these calls? Refer https://github.com/snapshot-labs/snapshot.js/blob/master/src/utils/multicaller.ts#L25

Reply:
Hi @ChaituVR, I removed unnecessary promises and awaits from the block, also removed the promise and await under.